### PR TITLE
Project list structure completed

### PR DESCRIPTION
**Pull request by Hombre2014**
All the requirements have been implemented:
- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- - Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
  - description [string].
  - completed [bool].
  - index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

[Video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

- All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,35 +30,42 @@ In this project, you will build a simple HTML list of To Do tasks. The list will
...
+- [Node](https://nodejs.org/en/download/)
 
 ### Usage
 
-- N/A
+- Clone the repository using "git clone git@github.com:Hombre2014/ToDoList.git"
+- Change directory into the project folder
+- Run npm install
+- Run npm start
+- A new browser will open automatically with application loaded
```
Comment (mikethreels) [inline README.md]: - [ ] [OPTIONAL] to make your readme file look even more professional you could put any commands in the command line or pieces of code in a code block by putting the peace of code between tilde (backtick) signs. I've added an image showing where it is located on your keyboard.
tilde

**Comment by mikethreels** [comment]
Hi Yuriy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] There is a merge conflict between the base branch and the feature branch, there are also no commit messages related to the update in this project, in order to get your project reviewed make sure that there are no conflicts, the linters are passing and there are descriptive commit messages related to this part of the project. Please send your project for review as soon as these issues are solved.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Yuriy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] There is a merge conflict between the base branch and the feature branch, there are also no commit messages related to the update in this project, in order to get your project reviewed make sure that there are no conflicts, the linters are passing and there are descriptive commit messages related to this part of the project. Please send your project for review as soon as these issues are solved.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Hombre2014** [comment]
Thanks for your comment and suggestions. I resolved the merger conflict and now the pull request is ready.

**Review by mikethreels — APPROVED** [review]
Hi Yuriy, 

Your project is complete! it's time to merge it 🚀 :
Congratulations! 🎉 

- [ ] [OPTIONAL] Because there are no changed files I'll put the comment here. to make your code look a bit cleaner and easier to read you might want to use the `.foreach` method instead of your own for-loop.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**